test/helpers/assert-valid.js

```diff
@@ -21,7 +20,7 @@ function formatError(error, doc, file) {
     const nodeType = lodash.get(doc, subPath) || '';
     const indent = i === 0 ? '' : '   '.repeat(i - 1) + '└─';
     if (nodeType === 'SourceUnit') c = file;
-    return lodash.compact([indent, nodeType, c && chalk.dim(c)]).join(' ');
+    return lodash.compact([indent, nodeType, c && util.styleText('dim', c).dim(c)]).join(' ');
```

Comment:
Should the `.dim(c)` part be removed since `styleText` already applies the format?
```suggestion
    return lodash.compact([indent, nodeType, c && util.styleText('dim', c)]).join(' ');
```

Reply:
Yes, I messed up the commit. However, this API is only available in Node 20 so I'll undo this change.